PythonTool/MakeImagesTerminal.py

```diff
@@ -0,0 +1,144 @@
+                                 usage="Create plots to save.",
+                                 )
+
+parser.add_argument("-ds", "--dataconvertstl",
+                    type=str,
+                    help="Convert h5m to stl.",
+                    )
+parser.add_argument("-dv", "--dataconvertvtk",
+                    type=str,
+                    help="Convert h5m to vtk.",
+                    )
+parser.add_argument("-gr", "--graveremove",
+                    type=str,
+                    help="Remove grave yard from mesh.",
+                    )
+parser.add_argument("-su", "--surfaces",
+                    type=str,
+                    help="Create curve file from h5m file.",
+                    )
+parser.add_argument("-cu", "--curves",
+                    type=str,
+                    help="Create curve file from h5m file.",
+                    )
+parser.add_argument("-pl", "--plots",
+                    type=str,
+                    help="List of plots.",
+                    )
+parser.add_argument("-op", "--operators",
+                    type=str,
+                    help="List of operators.",
+                    )
+parser.add_argument("-im", "--images",
+                    action="store_true",
+                    help="Make images of plots and operators.",
+                    )
+parser.add_argument("-wi", "--windows",
+                    action="store_true",
+                    help="Open multiple windows.",
+                    )
+parser.add_argument("-de", "--default",
+                    action="store_true",
+                    help="Apply default operators.",
+                    )
+parser.add_argument("-mu", "--multislice",
+                    type=str,
+                    help="Apply only slices.",
+                    )
+parser.add_argument("-or", "--orbit",
+                    type=str,
+                    help="Gather orbital view (vertical/horizontal/both).",
+                    )
+parser.add_argument("-se", "--sessionreplacesingle",
+                    type=str,
+                    help="Replace loaded data from a session file.",
+                    )
+parser.add_argument("-sm", "--sessionreplacemultiple",
+                    type=str,
+                    help="Replace loaded data from multiple session files.",
+                    )
+parser.add_argument("-gi", "--grabimagesfromsessions",
+                    action="store_true",
+                    help="Load sessions and grab window images.",
+                    )
+
+args = parser.parse_args()
+
+# Gather plot and operator inputs.
+if args.plots:
+    # Imported here to avoid loading the viewer when not needed.
+    from VisitOptions import VisitOptions
+    FilePlots = ast.literal_eval(args.plots)
+
+    if args.operators:
+        OperatorSet = ast.literal_eval(args.operators)
+        Options = VisitOptions(args, FilePlots, OperatorSet)
+    else:
+        Options = VisitOptions(args, FilePlots)
+
+if args.dataconvertstl:
```

Comment:
Regarding 1:
Understood.

Regarding 2:
I currently have a script to extract the curves without any graveyard geometry (Curves.sh). I can quickly add an option to get the curves directly from an h5m file without any graveyard filtering. It would be a bash script that calls mbconvert for creating a vtk file of curves. Should I do it?